### PR TITLE
Update Dependency Policy

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
Mostly as discussed in the dev chat.

Additionally makes dependabot also update the github actions dependencies - b/c why not … 

based on asyncio for now to make changes visible

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "friday"
+
+  # Updates the dependencies of the GitHub Actions workflows
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```
Comment (harshil21) [inline .github/dependabot.yml:13]: why not weekly like `pip`?

Comment (Bibo-Joshi) [reply .github/dependabot.yml:13]: thought it to be less important that way it may reduce the traffic a bit. but I'm open to setting it to weekly as well

Comment (harshil21) [reply .github/dependabot.yml:13]: okay so dependabot will update `requirements.txt` for `pip` and will update which file for `github-actions`? Can we also make it update `pre-commit-config`?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ on:
     branches: 
       - master
       - v14
+  schedule:
+    # Run every night on 03:07 - odd time to spread load on GitHub Actions
```
Comment (harshil21) [inline .github/workflows/test.yml:12]:
```suggestion
    # Run every night at 03:07 - odd time to spread load on GitHub Actions
```

File: .github/dependabot.yml (hunk 1 above)
Comment (Bibo-Joshi) [reply .github/dependabot.yml:13]: Yes, it will even track more files - [this table] lists `pipfile(.lock)` and `setup.py` and for [my repo](https://github.com/Bibo-Joshi/led-strip-controller/network/updates), the insights/dep graph/dependabot also lists `pyproject.toml`, `reqs-dev.txt` and `setup.cfg`.

for `pre-commit` I don't find anything in the [supported ecosystems](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems). We could consider switching to the pre-commit.ci for the `pre-commit` tests - maybe another item for #2925 :D

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -8,6 +8,9 @@ on:
     branches: 
       - master
       - v14
+  schedule:
+    # Run every night at 03:07 - odd time to spread load on GitHub Actions
+    - cron: '7 3 * * *'
```
Comment (Poolitzer) [inline .github/workflows/test.yml:13]: Sorry, whats the point of running the whole pytest suit every night?

File: requirements.txt [hunk 4]
```diff
@@ -1,8 +1,21 @@
...
 httpx ~= 0.22.0
 # only telegram.ext: # Keep this line here; used in setup(-raw).py
-tornado>=6.1
-APScheduler==3.8.1
+
+# tornado is rather stable, but let's not allow the next mayor release without prior testing
+tornado~=6.1
+
+# Cachetools and APS don't have a strict stability policy.
+# Let's be cautious for.
```
Comment (Poolitzer) [inline requirements.txt:16]: cautious for what?

**Review by Poolitzer — CHANGES_REQUESTED** [review]
I have so many (2) questions

File: requirements.txt (hunk 4 above)
Comment (Bibo-Joshi) [reply requirements.txt:16]: for now - sorry

**Review by harshil21 — APPROVED** [review]

File: .github/workflows/test.yml (hunk 3 above)
Comment (Poolitzer) [reply .github/workflows/test.yml:13]: Would dependabot also trigger if the dependency gets a new version in our defined boundaries?

If yes, I get why this is theory differing from dependabot right now, but that feels like we do basically the same work twice. One setup changes the requirements weekly but then also tests the new versions, this tests the same changes if they happen in a bit of a smaller boundary, but still the same ones.

Maybe a setup were we just let dependabot notify us of a new release as soon as it happens, we see if the PR actions run through, and if they don't we know we need to change something?

Comment (Bibo-Joshi) [reply .github/workflows/test.yml:13]: I see your point and am open to relying on dependabot only. In that case, I would change to `schedule: daily` and also set the [max number of depandabot PR](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit) to a number notably larger than 5 - just in case …

If we're unhappy with how/when tests are running, we can always change that later on, too :)

Comment (Bibo-Joshi) [reply .github/workflows/test.yml:13]: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy

this seems to imply that dependabot would indeed trigger even if we only have lower bounds. I can try and double check later today

Comment (Bibo-Joshi) [reply .github/workflows/test.yml:13]: Just tested at https://github.com/Bibo-Joshi/python-telegram-bot and it seems like dependabot doesn't bump if we have a lower bound only.

We have lower bounds for cryptography, ujson and for pytz, all of which have newer versions available.

PRs that dependabot created: https://github.com/Bibo-Joshi/python-telegram-bot/pulls

Log from dependabot. interestingly, the logs show nothing about `ujson` at all …

<details>

```
 proxy | time="2022-04-28T16:19:00Z" level=info msg="proxy starting" commit=d0e8fc9c52e08bf359a8a4cff6deb91b01c23136
  proxy | 2022/04/28 16:19:00 Listening (:1080)
updater | 2022-04-28T16:19:01.091586117 [anonymous-instance:main:WARN:src/firecracker/src/main.rs:370] You are using a deprecated parameter: --seccomp-level 2, that will be removed in a future version.
updater | 2022-04-28T16:19:01.154436792 [354770423:main:WARN:src/devices/src/legacy/serial.rs:432] Detached the serial input due to peer close/error.
updater | time="2022-04-28T16:19:04Z" level=info msg="guest starting" commit=d54c1f07420d2f98b700854f969f92bffb6a9ded
updater | time="2022-04-28T16:19:04Z" level=info msg="starting job..." fetcher_timeout=5m0s job_id=354770423 updater_timeout=45m0s updater_version=0.182.4-f07385a7abf3db9bd7a354993e0deb56f137fc1c
updater | I, [2022-04-28T16:19:09.422464 #7]  INFO -- sentry: ** [Raven] Raven 3.1.2 ready to catch errors
updater | warning: parser/current is loading parser/ruby27, which recognizes2.7.6-compliant syntax, but you are running 2.7.5.
updater | Please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
updater | INFO <job_354770423> Starting job processing
  proxy | 2022/04/28 16:19:17 [002] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot
  proxy | 2022/04/28 16:19:17 [002] * authenticating github api request
  proxy | 2022/04/28 16:19:17 [002] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot
  proxy | 2022/04/28 16:19:17 [004] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/git/refs/heads/master
  proxy | 2022/04/28 16:19:17 [004] * authenticating github api request
  proxy | 2022/04/28 16:19:17 [004] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/git/refs/heads/master
  proxy | 2022/04/28 16:19:17 [006] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:17 [006] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [006] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [008] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/pyproject.toml?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [008] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [008] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/pyproject.toml?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [010] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/MANIFEST.in?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [010] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [010] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/MANIFEST.in?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [012] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/requirements-dev.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [012] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [012] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/requirements-dev.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [014] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/requirements.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [014] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [014] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/requirements.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [016] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/.github?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [016] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [016] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/.github?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [018] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/contrib?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [018] * authenticating github api request
  proxy | 2022/04/28 16:19:18 [018] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/contrib?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [020] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/docs?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:18 [020] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [020] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/docs?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [022] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/docs/requirements-docs.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [022] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [022] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/docs/requirements-docs.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [024] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/examples?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [024] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [024] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/examples?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [026] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/examples/LICENSE.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [026] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [026] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/examples/LICENSE.txt?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [028] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/telegram?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [028] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [028] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/telegram?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [030] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/tests?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [030] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [030] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/tests?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [032] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/setup.py?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [032] * authenticating github api request
  proxy | 2022/04/28 16:19:19 [032] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/setup.py?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [034] GET https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/setup.cfg?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
  proxy | 2022/04/28 16:19:19 [034] * authenticating github api request
  proxy | 2022/04/28 16:19:20 [034] 200 https://api.github.com:443/repos/Bibo-Joshi/python-telegram-bot/contents/setup.cfg?ref=9c84433d14e3133865b420a13d7f6c9d764aa011
updater | INFO <job_354770423> Finished job processing
updater | time="2022-04-28T16:19:20Z" level=info msg="task complete" container_id=job-354770423-file-fetcher exit_code=0 job_id=354770423 step=fetcher
updater | I, [2022-04-28T16:19:22.391794 #7]  INFO -- sentry: ** [Raven] Raven 3.1.2 ready to catch errors
updater | warning: parser/current is loading parser/ruby27, which recognizes2.7.6-compliant syntax, but you are running 2.7.5.
updater | Please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
updater | INFO <job_354770423> Starting job processing
updater | INFO <job_354770423> Starting update job for Bibo-Joshi/python-telegram-bot
updater | INFO <job_354770423> Checking if black 21.9b0 needs updating
  proxy | 2022/04/28 16:19:47 [038] GET https://pypi.org:443/simple/black/
  proxy | 2022/04/28 16:19:47 [038] 200 https://pypi.org:443/simple/black/
updater | INFO <job_354770423> Latest version is 22.3.0
updater | INFO <job_354770423> Pull request already exists for black with latest version 22.3.0
updater | INFO <job_354770423> Checking if flaky  needs updating
  proxy | 2022/04/28 16:19:48 [040] GET https://pypi.org:443/simple/flaky/
  proxy | 2022/04/28 16:19:48 [040] 200 https://pypi.org:443/simple/flaky/
updater | INFO <job_354770423> Latest version is 3.7.0
  proxy | 2022/04/28 16:19:48 [042] GET https://pypi.org:443/simple/flaky/
  proxy | 2022/04/28 16:19:48 [042] 200 https://pypi.org:443/simple/flaky/
updater | INFO <job_354770423> No update needed for flaky 
updater | INFO <job_354770423> Checking if httpx  needs updating
  proxy | 2022/04/28 16:19:48 [044] GET https://pypi.org:443/simple/httpx/
  proxy | 2022/04/28 16:19:48 [044] 200 https://pypi.org:443/simple/httpx/
updater | INFO <job_354770423> Latest version is 0.22.0
updater | INFO <job_354770423> No update needed for httpx 
updater | INFO <job_354770423> Checking if pytz  needs updating
  proxy | 2022/04/28 16:19:48 [046] GET https://pypi.org:443/simple/pytz/
  proxy | 2022/04/28 16:19:48 [046] 200 https://pypi.org:443/simple/pytz/
updater | INFO <job_354770423> Latest version is 2022.1
  proxy | 2022/04/28 16:19:49 [048] GET https://pypi.org:443/simple/pytz/
  proxy | 2022/04/28 16:19:49 [048] 200 https://pypi.org:443/simple/pytz/
updater | INFO <job_354770423> Requirements to unlock update_not_possible
updater | INFO <job_354770423> Requirements update strategy bump_versions
updater | INFO <job_354770423> No update possible for pytz 
updater | INFO <job_354770423> Checking if sphinx 4.4.0 needs updating
  proxy | 2022/04/28 16:19:49 [050] GET https://pypi.org:443/simple/sphinx/
  proxy | 2022/04/28 16:19:49 [050] 200 https://pypi.org:443/simple/sphinx/
updater | INFO <job_354770423> Latest version is 4.5.0
updater | INFO <job_354770423> Pull request already exists for sphinx with latest version 4.5.0
updater | INFO <job_354770423> Checking if tornado  needs updating
  proxy | 2022/04/28 16:19:49 [052] GET https://pypi.org:443/simple/tornado/
  proxy | 2022/04/28 16:19:49 [052] 200 https://pypi.org:443/simple/tornado/
updater | INFO <job_354770423> Latest version is 6.1
updater | INFO <job_354770423> No update needed for tornado 
updater | INFO <job_354770423> Checking if furo 2022.1.2 needs updating
  proxy | 2022/04/28 16:19:50 [054] GET https://pypi.org:443/simple/furo/
  proxy | 2022/04/28 16:19:50 [054] 200 https://pypi.org:443/simple/furo/
updater | INFO <job_354770423> Latest version is 2022.4.7
updater | INFO <job_354770423> Pull request already exists for furo with latest version 2022.4.7
updater | INFO <job_354770423> Checking if click 8.0.2 needs updating
  proxy | 2022/04/28 16:19:50 [056] GET https://pypi.org:443/simple/click/
  proxy | 2022/04/28 16:19:50 [056] 200 https://pypi.org:443/simple/click/
updater | INFO <job_354770423> Latest version is 8.1.2
updater | INFO <job_354770423> Pull request already exists for click with latest version 8.1.2
updater | INFO <job_354770423> Checking if pylint 2.12.1 needs updating
  proxy | 2022/04/28 16:19:50 [058] GET https://pypi.org:443/simple/pylint/
  proxy | 2022/04/28 16:19:50 [058] 200 https://pypi.org:443/simple/pylint/
updater | INFO <job_354770423> Latest version is 2.13.7
updater | INFO <job_354770423> Pull request already exists for pylint with latest version 2.13.7
updater | INFO <job_354770423> Checking if pytest 6.2.5 needs updating
  proxy | 2022/04/28 16:19:50 [060] GET https://pypi.org:443/simple/pytest/
  proxy | 2022/04/28 16:19:50 [060] 200 https://pypi.org:443/simple/pytest/
updater | INFO <job_354770423> Latest version is 7.1.2
updater | INFO <job_354770423> Pull request already exists for pytest with latest version 7.1.2
updater | INFO <job_354770423> Checking if pytest-asyncio 0.16.0 needs updating
  proxy | 2022/04/28 16:19:51 [062] GET https://pypi.org:443/simple/pytest-asyncio/
  proxy | 2022/04/28 16:19:51 [062] 200 https://pypi.org:443/simple/pytest-asyncio/
updater | INFO <job_354770423> Latest version is 0.18.3
updater | INFO <job_354770423> Pull request already exists for pytest-asyncio with latest version 0.18.3
updater | INFO <job_354770423> Checking if mypy 0.910 needs updating
  proxy | 2022/04/28 16:19:51 [064] GET https://pypi.org:443/simple/mypy/
  proxy | 2022/04/28 16:19:51 [064] 200 https://pypi.org:443/simple/mypy/
updater | INFO <job_354770423> Latest version is 0.950
updater | INFO <job_354770423> Pull request already exists for mypy with latest version 0.950
updater | INFO <job_354770423> Checking if beautifulsoup4  needs updating
  proxy | 2022/04/28 16:19:52 [066] GET https://pypi.org:443/simple/beautifulsoup4/
  proxy | 2022/04/28 16:19:52 [066] 200 https://pypi.org:443/simple/beautifulsoup4/
updater | INFO <job_354770423> Latest version is 4.11.1
  proxy | 2022/04/28 16:19:52 [068] GET https://pypi.org:443/simple/beautifulsoup4/
  proxy | 2022/04/28 16:19:52 [068] 200 https://pypi.org:443/simple/beautifulsoup4/
updater | INFO <job_354770423> No update needed for beautifulsoup4 
updater | INFO <job_354770423> Checking if cryptography  needs updating
  proxy | 2022/04/28 16:19:53 [070] GET https://pypi.org:443/simple/cryptography/
  proxy | 2022/04/28 16:19:53 [070] 200 https://pypi.org:443/simple/cryptography/
updater | INFO <job_354770423> Latest version is 37.0.1
  proxy | 2022/04/28 16:19:54 [072] GET https://pypi.org:443/simple/cryptography/
  proxy | 2022/04/28 16:19:54 [072] 200 https://pypi.org:443/simple/cryptography/
updater | INFO <job_354770423> Requirements to unlock update_not_possible
updater | INFO <job_354770423> Requirements update strategy bump_versions
updater | INFO <job_354770423> No update possible for cryptography 
updater | INFO <job_354770423> Checking if pyupgrade 2.29.0 needs updating
  proxy | 2022/04/28 16:19:56 [074] GET https://pypi.org:443/simple/pyupgrade/
  proxy | 2022/04/28 16:19:56 [074] 200 https://pypi.org:443/simple/pyupgrade/
updater | INFO <job_354770423> Latest version is 2.32.0
updater | INFO <job_354770423> Pull request already exists for pyupgrade with latest version 2.32.0
updater | INFO <job_354770423> Checking if flake8 4.0.1 needs updating
  proxy | 2022/04/28 16:19:56 [076] GET https://pypi.org:443/simple/flake8/
  proxy | 2022/04/28 16:19:56 [076] 200 https://pypi.org:443/simple/flake8/
updater | INFO <job_354770423> Latest version is 4.0.1
updater | INFO <job_354770423> No update needed for flake8 4.0.1
updater | INFO <job_354770423> Checking if apscheduler  needs updating
  proxy | 2022/04/28 16:19:56 [078] GET https://pypi.org:443/simple/apscheduler/
  proxy | 2022/04/28 16:19:56 [078] 200 https://pypi.org:443/simple/apscheduler/
updater | INFO <job_354770423> Latest version is 3.9.1
updater | INFO <job_354770423> No update needed for apscheduler 
updater | INFO <job_354770423> Checking if sphinx-pypi-upload  needs updating
  proxy | 2022/04/28 16:19:56 [080] GET https://pypi.org:443/simple/sphinx-pypi-upload/
  proxy | 2022/04/28 16:19:56 [080] 200 https://pypi.org:443/simple/sphinx-pypi-upload/
updater | INFO <job_354770423> Latest version is 0.2.1
  proxy | 2022/04/28 16:19:57 [082] GET https://pypi.org:443/simple/sphinx-pypi-upload/
  proxy | 2022/04/28 16:19:57 [082] 200 https://pypi.org:443/simple/sphinx-pypi-upload/
updater | INFO <job_354770423> No update needed for sphinx-pypi-upload 
updater | INFO <job_354770423> Checking if cachetools  needs updating
  proxy | 2022/04/28 16:19:57 [084] GET https://pypi.org:443/simple/cachetools/
  proxy | 2022/04/28 16:19:57 [084] 200 https://pypi.org:443/simple/cachetools/
updater | INFO <job_354770423> Latest version is 5.0.0
updater | INFO <job_354770423> No update needed for cachetools 
updater | INFO <job_354770423> Checking if pre-commit  needs updating
  proxy | 2022/04/28 16:19:57 [086] GET https://pypi.org:443/simple/pre-commit/
  proxy | 2022/04/28 16:19:57 [086] 200 https://pypi.org:443/simple/pre-commit/
updater | INFO <job_354770423> Latest version is 2.18.1
  proxy | 2022/04/28 16:19:57 [088] GET https://pypi.org:443/simple/pre-commit/
  proxy | 2022/04/28 16:19:57 [088] 200 https://pypi.org:443/simple/pre-commit/
updater | INFO <job_354770423> No update needed for pre-commit 
updater | INFO <job_354770423> Checking if wheel  needs updating
  proxy | 2022/04/28 16:19:57 [090] GET https://pypi.org:443/simple/wheel/
  proxy | 2022/04/28 16:19:57 [090] 200 https://pypi.org:443/simple/wheel/
updater | INFO <job_354770423> Latest version is 0.37.1
  proxy | 2022/04/28 16:19:58 [092] GET https://pypi.org:443/simple/wheel/
  proxy | 2022/04/28 16:19:58 [092] 200 https://pypi.org:443/simple/wheel/
updater | INFO <job_354770423> No update needed for wheel 
updater | INFO <job_354770423> Finished job processing
updater | time="2022-04-28T16:19:58Z" level=info msg="task complete" container_id=job-354770423-updater exit_code=0 job_id=354770423 step=updater
```

</details>

Comment (Poolitzer) [reply .github/workflows/test.yml:13]: weird. But if we aren't willing to ask if thats intended/ want to merge this, I would be alright if we do it like this, but maybe like 2 times per week is enough? daily feels a bit excessive

Comment (Bibo-Joshi) [reply .github/workflows/test.yml:13]: I don't think that we get a dependabot upgrade for that within a week 😄 

> maybe like 2 times per week is enough? daily feels a bit excessive

that's fine with me, I guess. Maybe mondays and fridays?

Comment (Poolitzer) [reply .github/workflows/test.yml:13]: sure, good days.

**Review by Poolitzer — APPROVED** [review]
LGTM
